pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -151,7 +151,8 @@ func (actuator *OvirtActuator) Exists(_ context.Context, machine *machinev1.Mach
 	}
 	vm, err := machineService.GetVm(*machine)
 	if err != nil {
-		return false, err
+		klog.Warningf("oVirt VM %s does not exist", machine.Name)
+		return false, nil
```

Comment:
This may not be true, what if there was a connection error for example? You need to check here if the error coming back from the Machine service is a "Not Found/404" error or something else before asserting this

Reply:
you are very right, I'll send a fix